.github/scripts/gh_ci_entry.sh

```
@@ -0,0 +1,194 @@
+
+# --- Download & start Minio ---
+echo "Downloading and starting Minio..."
+wget -q https://dl.min.io/server/minio/release/linux-amd64/minio
+chmod +x minio
+mkdir -p /tmp/minio_data
+MINIO_ROOT_USER=$MINIO_ACCESS_KEY MINIO_ROOT_PASSWORD=$MINIO_SECRET_KEY \
+  ./minio server /tmp/minio_data --address :9000 --console-address :9001 > /tmp/minio.log 2>&1 &
+MINIO_PID=$!
+
+echo "Waiting for Minio to be ready..."
+for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)